qutebrowser/keyinput/basekeyparser.py

```diff
@@ -148,7 +172,16 @@ def _match_count(self, sequence, dry_run):
             return True
         return False
 
-    def handle(self, e, *, dry_run=False):
+    def _get_url(self) -> typing.Optional[QUrl]:
+        try:
+            tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                        window='current')
+            return tabbed_browser.current_url()
+        except objreg.RegistryUnavailableError:
+            return None
+
+    def handle(self, e: QEvent, *, dry_run: bool = False,
+               url: typing.Optional[QUrl] = None):
```

Comment:
I couldn't find a way to stub in a `url` without passing in a function (`tabbed_browser_stubs` didn't seem to propogate to objreg). Do you know if there's a better way to do that (or is this better than the alternatives?)